Classes/UT3Manta.uc

```diff
@@ -42,6 +42,8 @@
 class UT3Manta extends ONSHoverBike;
 
 var Emitter DuckEffect;
+Var rotator ThighDriveL,ThighDriveR;
```

Comment:
This can be put between lines 220 and 221 as
```UnrealScript
local rotator ThighDriveL,ThighDriveR;
```
Alternatively, you can remove lines 224 and 226 and instead add to `defaultproperties`:
```UnrealScript
ThighDriveL = (Pitch=1800)
ThighDriveR = (Pitch=-1800)
```